lib/platform.test.js

```diff
@@ -170,4 +172,31 @@ describe('Platform', () => {
 			expect(p.hasNcpFirmware).to.be.false;
 		});
 	});
+
+	describe('electron2', () => {
+		it('has electron2 platform definitions by id', () => {
+			const p = platformForId(37);
+			const esomx = platformForName('esomx');
```

Comment:
I think this line `const esomx = platformForName('esomx');` is not necessary in electron2 context
Also this one `expect(esomx).to.not.be.null;` 

I'm reviewing this since I was doing the same thing (updating device-constants over here 👉  https://github.com/particle-iot/device-os-flash-util/compare/master...chore/sc-132686/support-electron-2-platform-in-cli-tools)

Reply:
While not necessary, it's an easy way to validate that Electron 2 platform definition is the same as its close sibling: ESoMX.